libs/isograph-react/src/react/useReadAndSubscribe.ts

```diff
@@ -35,6 +36,13 @@ export function useReadAndSubscribe<
     setReadOutDataAndRecords,
     readerAst,
   );
+
+  if (readOutDataAndRecords.errors) {
+    throw new GraphqlAggregateError(
+      readOutDataAndRecords.errors.map((error) => new GraphqlError(error)),
```

Comment:
It might be nice to not create new errors on every render, but instead to have stable references here.